### DISCUSSION TITLE
Possible way to add a language dropdown field in "general"?

### DISCUSSION THREAD

**Discussion by lars-s-k** [gotson/komga#1772] [Q&A] [answered]
Hey together,

after I changed now from ComicRack to Komga, the programer(s) have done a great job 👍 .
One question I have, is there a possible way of a language dropdown field? Because I have German and English comics and would than to filter them afte the language option; or is the only way to "tag" them in the "tag field".
Thanks in advance and nice greetings

**Comment by gotson** [comment, marked as answer]
There's a language metadata field at series level, then you can filter by language.

**Reply by lars-s-k** [reply]
Thanks for explaination and I not expected it there, I expected it direct at the comic.
Lessons learned  👍 
Nice greetings
